packages/astro/src/assets/fonts/utils.ts

```diff
@@ -54,6 +59,12 @@ export function renderFontSrc(sources: Exclude<unifont.FontFaceData['src'][numbe
 		.join(', ');
 }
 
+const QUOTES_RE = /^["']|["']$/g;
```

Comment:
What's "quotes re"?

Reply:
It allows removing start and end quotes if they are present AFAIK